src/main/java/net/sf/jabref/logic/logging/GuiAppender.java

```diff
@@ -37,10 +39,25 @@ public static GuiAppender createAppender(@PluginAttribute("name") String name,
         return new GuiAppender(name, filter, layout);
     }
 
-
+    /**
+     * The log event will be forwarded to the {@link LogMessages} archive.
+     * <p/>
+     * All logged messages will pass this method. For each line in the messages of the log event a new log event will be created.
+     * @param event log event
+     */
     @Override
     public void append(LogEvent event) {
-        CACHE.add(new String(this.getLayout().toByteArray(event)));
+        String message = new String(this.getLayout().toByteArray(event));
+        message = message.replace("\n\r", "\n").replace("\r", "\n");
+        // stack traces logged by 'Log.error("message"), e' will be split by new lines so we can create a new log event for each line as 'e.printStackTrace()' would do.
+        if (event.getLevel() == Level.ERROR) {
+            Arrays.asList(message.split("\n")).stream().filter(s -> !s.isEmpty()).forEach(log -> {
+                LogEvent messageWithPriority = Log4jLogEvent.newBuilder().setMessage(new SimpleMessage(log.replace("\n", ""))).setLevel(event.getLevel()).build();
```

Comment:
@tobiasdiez if i don't split this it will be show me not so nice in list view. It looks like this, because in append-method of GuiAppender I get only one LogEvent entry with all information by insert a log entry like Log.error (e). It is not like System.err:

![2016-09-19_07h12_28](https://cloud.githubusercontent.com/assets/13031192/18623035/70c3dbba-7e38-11e6-803e-1e4b61f12877.png)
And if i don't replace System.lineseparator with "" then i will get 2 line each list cell.

That is the reason, why i have reformat this log event entry. That look better. 😄